src/main/java/edu/usf/cutr/pelias/SearchRequest.java

```diff
@@ -144,11 +144,11 @@ public Builder setApiEndpoint(String apiEndpoint) {
          * Builds the SearchRequest using the specified parameters
          * @return the SearchRequest using the specified parameters
          */
-        public SearchRequest build() {
+        public SearchRequest build() throws java.io.UnsupportedEncodingException {
             StringBuilder builder = new StringBuilder();
             builder.append(mApiEndPoint);
             builder.append("?text=");
-            builder.append(mText);
+            builder.append( URLEncoder.encode(mText, "UTF-8") );
```

Comment:
Thinking more,  it would probably make more sense to encode the text in the constructor,  and throw the exception there.   That's a more relevant  context for the library user,  than waiting until building the request.